system/lib/wasmfs/js_api.cpp

```diff
@@ -232,6 +232,18 @@ int _wasmfs_close(int fd) {
   return __wasi_fd_close(fd);
 }
 
+int _wasmfs_mmap(size_t length, int prot, int flags, int fd, size_t offset) {
+  return __syscall_mmap2(0, length, prot, flags, fd, offset);
```

Comment:
Looks like we have `position` in JS and `offset` in c++ here? If the name here is due to the syscall, let's rename the JS perhaps?

(If the reason for the JS name was to match the JS FS API, that's a valid reason too, but a less important one as that code will go away eventually.)

Reply:
Now that I look at it again, `__syscall_mmap2` takes in `size_t off`, while the old JS FS API took in `position`. Would it be best to rename the JS to `offset` to match the syscall?

Reply:
Yes, `offset` seems best to me.